worldwind/build.gradle

```diff
@@ -123,8 +123,8 @@ task compileJavadocSources(type: Javadoc) {
     description 'Creates the Javadoc API documentation'
     source android.sourceSets.main.java.srcDirs
     include('**/*.java')
-    title 'NASA World Wind Android'
-    options.windowTitle 'NASA World Wind Android'
+    title 'NASA WorldWind Android'
```

Comment:
Note: These JavaDoc arguments affect the documentation displayed at https://worldwind.arc.nasa.gov/android/docs/